test/cypress/support/commands.js

```diff
@@ -352,13 +355,16 @@ Cypress.Commands.add('giCreateGroup', (name, {
           const eventHandler = ({ groupContractID }) => {
             if (groupContractID === cID) {
               sbp('okTurtles.events/off', JOINED_GROUP, eventHandler)
-              const invervalId = setInterval(() => {
+              sbp('chelonia/contract/retain', groupContractID, { ephemeral: true }).then(() => sbp('chelonia/contract/sync', groupContractID)).then(() => sbp('okTurtles.eventQueue/queueEvent', 'event-handled', Boolean)).then(() => {
```

Comment:
This is a long line with correspondingly long comments.

> `sbp('okTurtles.eventQueue/queueEvent', 'event-handled', Boolean)`

This looks quite understandable to me, but I see the issue. Think of `Boolean` as `() => {}`. I.e., this is pushing to the end of the queue and waiting. The actual thing that matters is waiting.

> Let's say no longer than 130 columns max.

Agreed! We've had this conversation before and I've tried to keep lines short, but sometimes like here they slip up. I'll fix this, but as I noted then it'd save everyone's time to do this automatically.

> It wasn't there before and tests were passing... so why is it needed now?

Well, tests weren't entirely passing before, or rather there were race conditions that are now exacerbated by having the SW asynchronously handling events.

Calling `retain` / `release` is the proper way to `sync` a contract, because it guarantees that it won't be removed.